test/runtests.py

```diff
@@ -104,10 +103,10 @@
 tags = set(args.tag or [])
 not_tags = set(args.not_tag or []).union(['fail', 'exclude_' + arch])
 
-if arch_alias:
+if arch:
     not_tags.add('exclude_' + arch_alias)
-if flavor_alias:
-    not_tags.add('exclude_' + flavor_alias)
+if flavor:
+    not_tags.add('exclude_' + flavor)
```

Comment:
Thanks @obastemur.
There is `not_tags = lower_set(not_tags)` in runtests.py, it will convert `exclude_Test` to `exclude_test`. I tried to run tests in my ubuntu box by execution runtests.py for both test&debug configration, it will all pass. I doubt the issue is in runtests.sh, but it is used for CI. I can't running successful in local.